journey/src/main/kotlin/com/pingidentity/journey/callback/MetadataCallback.kt

```diff
@@ -80,11 +124,10 @@ class MetadataCallback : AbstractCallback() {
         if (value[ACTION]?.jsonPrimitive?.contentOrNull == WEBAUTHN_AUTHENTICATION) {
             return true
         }
-
-        // Checking for existence and content of _TYPE and either PUB_KEY_CRED_PARAMS
-        // or _PUB_KEY_CRED_PARAMS
+        // Checking for existence and content of _TYPE and not with PUB_KEY_CRED_PARAMS
+        // and _PUB_KEY_CRED_PARAMS
         return value[TYPE]?.jsonPrimitive?.contentOrNull == WEB_AUTHN &&
-                (value.containsKey(PUB_KEY_CRED_PARAMS) || value.containsKey(_PUB_KEY_CRED_PARAMS))
+                !value.containsKey(PUB_KEY_CRED_PARAMS) && !value.containsKey(_PUB_KEY_CRED_PARAMS)
```

Comment:
I would suggest making that a bit more explicit. At the moment we are checking that certain parameters are not there to determine the registration. For future proofing this and avoiding errors in logic I would suggest trying to match something that is registration specific like the allowed creds. Example:
```
return value[TYPE]?.jsonPrimitive?.contentOrNull == WEB_AUTHN &&
       (value.containsKey("_allowCredentials") || value.containsKey("allowCredentials"))
```